components/paging/pager-load-more.js

```diff
@@ -98,7 +97,7 @@ class LoadMore extends PageableSubscriberMixin(FocusMixin(LocalizeCoreElement(Rt
 			${this._loading ? html`
 				<d2l-loading-spinner size="24"></d2l-loading-spinner>
 			` : html`
-				<span class="action">${this.localize('components.pager-load-more.action', { count: formatNumber(this.pageSize) })}</span>
+				<span class="action">${isNaN(this.pageSize) ? this.localize('components.pager-load-more.action') : this.localize('components.pager-load-more-with-page-size.action', { count: formatNumber(this.pageSize) })}</span>
```

Comment:
I think all the terms for the pager component should be grouped by beginning with `components.pager-load-more.*`.